pkgs/development/python-modules/torch/gpu-checks.nix

```diff
@@ -13,7 +13,7 @@ let
       torch,
       cudaPackages,
     }:
-    cudaPackages.writeGpuPythonTest
+    cudaPackages.writeGpuTestPython
```

Comment:
Oh that's a silly one... thanks. Definitely need to have Ofborg at least evaluate these